### PR TITLE
Update admin image purge tool to purge images from Fastly

### PR DESCRIPTION
**Pull request by philmcmahon**
## What does this change?
This modifies the admin purge tool, removing any imgix specific code and adding the new fastly image service.

## What is the value of this and can you measure success?
We will be able to purge images from our new image service 

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by paperboyo** [comment]
👏 
Both masters (and all their renders) and specific renders through the same tool?

**Comment by philmcmahon** [comment]
This version will remove all renders and the master - query parameters are ignored

File: admin/app/controllers/cache/ImageServices.scala [hunk 1]
```diff
@@ -2,53 +2,31 @@ package controllers.cache
...
 
-class ImageServices(wsClient: WSClient) {
+object ImageServices {
 
   // none of the stuff here is a state secret.
   // it is all authenticated
 
-  private val iGuimCoUk = "YjEge7vUbdglYXtZ56cU1"
+  private val iGuimCoUk = "1L0HRheo6sMtfQHnY1FU6C"
+  private val iGuimCodeCoUk = "5CSDV7WcKwnIIHipZzt3po"
```
Comment (shtukas) [inline admin/app/controllers/cache/ImageServices.scala]: I would rename ``iGuimCoUk and `iGuimCodeCoUk` :)

File: admin/app/views/cache/imageDecache.scala.html [hunk 2]
```diff
@@ -16,7 +16,9 @@ <h1>Clear image</h1>
                 </code>
             </p>
             <p>
-                Works with images on <code>i.guim.co.uk</code>, <code>static.guim.co.uk</code> and <code>media.guim.co.uk</code>
+                Works with images on <code>i.guim.co.uk</code>, <code>static.guim.co.uk</code> and <code>media.guim.co.uk</code>.
```
Comment (shtukas) [inline admin/app/views/cache/imageDecache.scala.html]: Do we need `i.guim.co.uk` ?

**Review by shtukas — APPROVED** [review]

File: admin/app/controllers/cache/ImageServices.scala (hunk 1 above)
Comment (philmcmahon) [reply admin/app/controllers/cache/ImageServices.scala]: Do you mean to fastlyioguimcouk or something similar? That domain should be deprecated soon, now that we have migrated i.guim over to it. fastly-io-code is still in use, but I've moved i.guimcode over to that service so we can stop using that too.

**Pull request merged**